src/compas_rhino/install.py

```diff
@@ -202,6 +204,9 @@ def install(version=None, packages=None, clean=False):
     if exit_code != 0:
         sys.exit(exit_code)
 
+    global INSTALLED_VERSION
```

Comment:
@gonzalocasas @brgcode how about something like this?

Reply:
sounds good to me